### PR TITLE
Debugger: render code snipet in CLI on error (#494)(#490)

### PR DESCRIPTION
**Pull request by janbarasek**
- new feature
- BC break? no

Rel #490

A simple tool for plotting an uncaught exception (or other error) directly to the terminal in CLI mode. It makes it very easy to debug the application on the server.

Real example:

Screenshot from 2021-06-09 09-24-11

Thanks!

### CODE REVIEW

File: src/Tracy/Debugger/Debugger.php [hunk 1]
```diff
@@ -323,7 +323,9 @@ public static function exceptionHandler(\Throwable $exception): void
 				@fwrite(STDERR, "ERROR: {$exception->getMessage()}\n"
 					. (isset($e)
 						? 'Unable to log error. You may try enable debug mode to inspect the problem.'
-						: 'Check log to see more info.')
+						: 'Check log to see more info.') . "\n\n"
```
Comment (dg) [inline src/Tracy/Debugger/Debugger.php:326]: To by si mělo vypisovat spíš jen v debug režimu, ne?

Comment (janbarasek) [reply src/Tracy/Debugger/Debugger.php:326]: CLI je vždy produkční, ne?

Comment (dg) [reply src/Tracy/Debugger/Debugger.php:326]: Ne, je to nezávislé.

Comment (janbarasek) [reply src/Tracy/Debugger/Debugger.php]: Doplněno.

File: src/Tracy/Debugger/Debugger.php [hunk 2]
```diff
@@ -324,7 +324,12 @@ public static function exceptionHandler(\Throwable $exception): void
 					. (isset($e)
 						? 'Unable to log error. You may try enable debug mode to inspect the problem.'
 						: 'Check log to see more info.')
-					. "\n");
+					. (!self::$productionMode
```
Comment (dg) [inline src/Tracy/Debugger/Debugger.php:327]: Did you test it?

Comment (janbarasek) [reply src/Tracy/Debugger/Debugger.php]: I think now it is perfect. First call is production, second is debug.

Snímek obrazovky 2021-06-09 v 20 04 29

Testing code:

```php
Debugger::$productionMode = false;
throw new \RuntimeException('Test for DG.');
```

Thanks.

**Comment by dg** [comment]
I'm gonna try to tweak it a little more.

**Pull request merged**